recipe/meta.yaml

```diff
@@ -1,37 +1,46 @@
-{% set version = "0.4.5" %}
+{% set version = "0.4.6" %}
 
 package:
   name: cons
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/cons/cons-{{ version }}.tar.gz
-  sha256: b46b48adb5a5af7f44375da346d926e55a325d4dc12b9add9f20280d3b3742cb
+  sha256: 669fe9d5ee916d5e42b9cac6acc911df803d04f2e945c1604982a04d27a29b47
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - logical-unification >=0.4.1
```

Comment:
should be >= 0.4.0 ? https://github.com/pythological/python-cons/blob/bafb13f9f25da3b13deb2b58d9d0709a259ef697/setup.py#L12C5-L12C53

Reply:
good catch!